Add unknown icon code

mmhobi7 opened the pull request:
I was debating whether or not to add it to 76-94 but I stuck with what I did instead
SVGs are still needed

mmhobi7 committed:
Add unknown icon code

I was debating whether or not to add it to 76-94 but I stuck with what I did instead
SVGs are still needed

andrebrait reviewed (changes requested):
The code seems good, but I can't merge until the SVGs are present. I'm afraid I can't take a look into it until December, at best.

mmhobi7 commented:
https://github.com/luzfcb/prime-indicator/commit/ac5084da9089765f3927b93b1126827a95a45ef3

andrebrait commented:
Ok, good. I'll see when I can review this again and merge. Thanks for the image!

mmhobi7 committed:
Add Unknwon SVGs
https://github.com/luzfcb/prime-indicator/commit/ac5084da9089765f3927b93b1126827a95a45ef3
